examples/demo_formats.py

```
@@ -76,4 +76,4 @@ def main():
 
 
 if __name__ == "__main__":
-    main()
+    main()
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)